code/modules/events/event_container.dm

```diff
@@ -170,7 +170,8 @@ var/list/event_last_fired = list()
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Gravitational Anomaly",	/datum/event/anomaly/anomaly_grav,		200),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Revenant", 				/datum/event/revenant, 					150),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Swarmer Spawn", 			/datum/event/spawn_swarmer, 			150, is_one_shot = 1),
-		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Morph Spawn", 				/datum/event/spawn_morph, 				0, is_one_shot = 1)
+		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Morph Spawn", 				/datum/event/spawn_morph, 				0, is_one_shot = 1),
+		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Traders",	 				/datum/event/traders,	 				150)
```

Comment:
Generally major events are only supposed to happen once. Also, you don't want too many of htem because then some will get stuck at CentComm and some will be stuck on station. It also generally means that there's less likely to be enough supplies to go around.